test/integration/access.js

```diff
@@ -22,13 +22,13 @@ module.exports = function (createFn, setup, dismantle) {
       let repeatCustomer;
       let repeatCustomerInvoice;
 
-      beforeEach((done) => {
+      before((done) => {
```

Comment:
I had to do that change in every test file for restify tests to pass:

- Move `setup` and `serve` calls in `before` so they are only called once
- Only call `db.reset` in `beforeEach`